source/features/raw-file-link.tsx

```diff
@@ -10,17 +10,16 @@ const createRawUrl = (href: string): string => {
 };
 
 function addRawButtons(): void {
-	for (const anchor of select.all<HTMLAnchorElement>('.file-header .file-actions > div.BtnGroup [href*=\'/blob/\']:last-child')) {
+	for (const anchor of select.all<HTMLAnchorElement>('.file-header .file-actions > div > details > details-menu > a:nth-child(3)')) {
```

Comment:
This selector seems overly specific, it should still look for `[href*=\'/blob/\']` instead